src/lug/Window/Android/WindowImplAndroid.cpp

```diff
@@ -44,7 +44,8 @@ float WindowImpl::MapCenteredAxis(AInputEvent* event, int32_t axis) {
 }
 
 int32_t WindowImpl::HandleInput(lug::Window::Event& event, AInputEvent* androidEvent) {
-  // Engine* eng = (Engine*)app->userData;
+    //TODO : refactor
```

Comment:
Can you add more details? This will be useless in 1 months when we don't have any context